src/_app/render.tsx

```diff
@@ -3,25 +3,24 @@ import React from 'react';
 import type {
   ExtendedOptions,
   NextApp,
-  PageObject,
+  GenericPageObject,
   PageProps,
 } from '../commonTypes';
-import { getPageComponents } from '../makePageElement';
 
 export default function renderApp({
   options,
   pageObject,
   pageProps,
 }: {
   options: ExtendedOptions;
-  pageObject: PageObject;
+  pageObject: GenericPageObject;
```

Comment:
Nit: having `Generic` in type name seems a bit redundant as `PageObject` seems generic enough. Probably means the "found" case should have a more detailed name.